### PR TITLE
feat(responsive-ui): Keep aspect ratio for filmstrip self view on mobile web

### PR DESCRIPTION
**Pull request by vp8x8**
Right now filmstrip displays self view in landscape mode.
With these changes the aspect ratio of the self view will be maintained
so on portrait mode the thumbnail will be displayed vertically.
Of course this makes sense only on mobile web.

Screenshot 2021-09-01 at 15 02 50
Screenshot 2021-09-01 at 15 03 04

### CODE REVIEW

**Review by horymury — CHANGES_REQUESTED** [review]

File: react/features/filmstrip/components/web/Thumbnail.js [hunk 1]
```diff
@@ -774,11 +783,14 @@ class Thumbnail extends Component<Props, State> {
         const { id } = _participant || {};
         const { audioLevel } = this.state;
         const styles = this._getStyles();
+        const localVideoStyle = {
+            ...styles.thumbnail,
+            height: _isMobilePortrait ? `${Math.floor(_height * MOBILE_FILMSTRIP_PORTRAIT_RATIO)}px` : styles.height
```
[the remark this answers is not in the document]
Comment (vp8x8) [reply react/features/filmstrip/components/web/Thumbnail.js]: Done

**Review by horymury — APPROVED** [review]

**Comment by vp8x8** [comment]
Jenkins test this please.

**Comment by vp8x8** [comment]
Jenkins please test this.

**Comment by vp8x8** [comment]
Jenkins please test this.

**Pull request merged**